### PR TITLE
make benchmark really working

### PR DESCRIPTION
**Pull request by panyx0718**
user complain they crash when following our doc.

### CODE REVIEW

File: benchmark/fluid/README.md [hunk 1]
```diff
@@ -29,9 +29,11 @@ Currently supported `--model` argument include:
     You can choose to use GPU/CPU training. With GPU training, you can specify
     `--gpus <gpu_num>` to run multi GPU training.
 * Run distributed training with parameter servers:
+    * see run_fluid_benchmark.sh as an example.
```
Comment (gongweibao) [inline benchmark/fluid/README.md]: Need a link here.

File: benchmark/fluid/README.md [hunk 2]
```diff
@@ -29,9 +29,11 @@ Currently supported `--model` argument include:
...
     * start parameter servers:
         ```bash
         PADDLE_TRAINING_ROLE=PSERVER PADDLE_PSERVER_PORT=7164 PADDLE_PSERVER_IPS=127.0.0.1 PADDLE_TRAINERS=1 PADDLE_CURRENT_IP=127.0.0.1 PADDLE_TRAINER_ID=0 python fluid_benchmark.py --model mnist  --device GPU --update_method pserver
```
Comment (gongweibao) [inline benchmark/fluid/README.md]: pserver runs on CPU

File: benchmark/fluid/run_fluid_benchmark.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+PADDLE_TRAINING_ROLE=PSERVER PADDLE_PSERVER_PORT=7164 PADDLE_PSERVER_IPS=127.0.0.1 PADDLE_TRAINERS=2 PADDLE_CURRENT_IP=127.0.0.1 PADDLE_TRAINER_ID=0 python fluid_benchmark.py --model resnet --device GPU --update_method pserver --iterations=10000 &
```
Comment (Yancey1989) [inline benchmark/fluid/run_fluid_benchmark.sh]: Seems this command would print all logs on the terminal, we can startup them as follows:
```bash
PADDLE_TRAINING_ROLE=PSERVER ... stdbuf -oL nohup python fluid_benchmark.py <args> 2>&1 > server.log &
```

And then users would check the logs in the `server.log` file.

File: benchmark/fluid/README.md (hunk 1 above)
Comment (panyx0718) [reply benchmark/fluid/README.md]: Not sure this is needed. link can be broken and the file is just in this folder

File: benchmark/fluid/README.md (hunk 2 above)
Comment (panyx0718) [reply benchmark/fluid/README.md]: Done

File: benchmark/fluid/run_fluid_benchmark.sh (hunk 3 above)
Comment (panyx0718) [reply benchmark/fluid/run_fluid_benchmark.sh]: I think it's fine to print out some logs to give user some feedback. There aren't many outputs

```
Pass 0, batch 162, loss [2.7855887 2.973915 ]
Pass 0, batch 162, loss [3.0754983 3.2426462]
Pass 0, batch 171, loss [3.4701207 4.438573 ]
Pass 0, batch 171, loss [3.7791452 3.3191109]
```

**Pull request merged**